packages/database/src/utilities/mapOverlays.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ */
+
+export const getMapOverlayGroupId = async (db, code) => {
```

Comment:
There's a function under `database/src/utilities/migration.js` called `codeToId` which does this generically across any table

Reply:
Thanks for that. It's a handy util to be aware of.